stylesheets/_font_stack.scss

```diff
@@ -35,10 +27,7 @@ $NTA-Light-Tabular:
        local("Helvetica");
 }
 
-$Helvetica-Regular:
-  "GDS-Logo",
-  sans-serif;
+$Helvetica-Regular: "GDS-Logo", sans-serif;
```

Comment:
I think it needs to stay below the `@font-face` declaration, but I guess both blocks could move up? (There's no reference to this within the toolkit, but it's used in a couple of apps for organisation logo styles).